### DISCUSSION TITLE
Question on zfs-check

### DISCUSSION THREAD

**Discussion by Simon4711** [psy0rz/zfs_autobackup#175] [General]
I tried out the zfs-check tool, and I don't know how to see this failure:

root@pvezfsdus:/home# zfs-check rpool/crypt/hamdus1/USBBackup/Crypt/vm-100-disk-0@hamdus1-20221130030011 --skip 9 --debug | ssh root@pve3ham "zfs-check USBBackup/Crypt/vm-100-disk-0@hamdus1-20221130030011 --check"
**Chunk 56320 failed: da39a3ee5e6b4b0d3255bfef95601890afd80709 EOF**

I think this is the last generated hash on the volume. I tried access to this zfs vol generating a new one from this checked snapshot in Proxmox, it's a RAW in a Windows VM. It seems to be o.k.

But why is this error reported on zfs-check?

**Comment by psy0rz** [comment]
Continue in #176 , seems like either a zfs-check bug or actual zfs bug :)
